tools/clean-abandonned-files.js

```diff
@@ -0,0 +1,26 @@
+const fs = require("fs");
+
+const ed = require("../ed/index.json");
+
+const subdirs = ["dfns", "css", "headings", "idl", "idlparsed", "links", "refs"];
+
+
+const removeExtension = f => {
+  const components = f.split(".");
+  return components.slice(0, components.length - 1).join(".");
+}
+
+function checkDir(path) {
+  const dir = fs.readdirSync(path);
+  for (let filename of dir) {
+    const subdir = path.split("/")[1];
+    if (!ed.results.find(spec => spec[subdir] === subdir + "/" + filename)) {
+      fs.unlinkSync(path + "/" + filename);
+    }
+  }
+}
+
+for (let dir of subdirs) {
+  checkDir("ed/" + dir);
+  checkDir("tr/" + dir);
```

Comment:
What is the rationale behind comparing everything to `ed/index.json`? I would have expected the check on `tr/` to use `tr/index.json`, where the info could perhaps be slightly different (be it only because the TR crawl is only done once per week.

Reply:
the rationale is me being  distracted :)